FirebasePerformance/Tests/TestApp/README.md

```diff
@@ -3,9 +3,13 @@
 ## Setup
 
 Download the `GoogleService-Info.plist` file from [Firebase Console](https://console.firebase.google.com/)
-(for whatever Firebase project you have or want to integrate the `dev-app`). For Autopush environment, store the
+(for whatever Firebase project you have or want to integrate the `dev-app`).
```

Comment:
Lets qualify the importance of bundleId matching. I think we can mention the bundle ID here.

Reply:
Great suggestions! Added the bundle IDs there.